experimental/dds/tree2/src/test/feature-libraries/editable-tree/mockData.ts

```diff
@@ -62,15 +59,15 @@ export const phonesSchema = builder.fieldNode(
 	"Test:Phones-1.0.0",
 	builder.sequence([
 		stringSchema,
-		int32Schema,
+		leaf.number,
```

Comment:
In general, where possible, why not use `builder.number` instead of `leaf.number`? Do you prefer this pattern more?

Reply:
Because there can be code elsewhere which also needs to refer to the same number schema (for example when building trees), and the builder might not be in scope, we have to sometimes use the leaf.number patterns. Its also the pattern we have to use for all non-leaf schema, and also what you get if you find and replace all the ussages of the old schema with leaf.number (since doing builder.number would fail if there was no SchemaBuilder named builder in scope). Basically leaf.number works everywhere, and is the same pattern other cases use, and builder.number only works when a builder is in scope and only for leaf nodes. Its also shorter, and has better doc comments.

Reply:
Really though, it doesn't matter much.